docs/how-to-guides/tracing/trace_span.md

```diff
@@ -0,0 +1,150 @@
+# Overview
```

Comment:
Use another name, this name will show in the left nav section

Reply:
Tracing span spec